ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/techpreview/provision/cucushift-installer-rehearse-vsphere-ipi-zones-techpreview-provision-chain.yaml

```diff
@@ -1,9 +1,18 @@
 chain:
   as: cucushift-installer-rehearse-vsphere-ipi-zones-techpreview-provision
   steps:
-  - chain: ipi-conf-vsphere-zones
+  - ref: ipi-conf-vsphere-check
+  - ref: ipi-conf-vsphere-vips
+  - ref: ipi-conf-vsphere-lb
+  - ref: ipi-conf-vsphere-dns
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-vsphere-zones
   - chain: ipi-install-vsphere
   - ref: enable-qe-catalogsource
+  env:
```

Comment:
@jinyunma also need FEATURE_SET for chain cucushift-installer-rehearse-aws-ipi-proxy-techpreview-po-provision? I think so.

@jianzhangbjz could you please confirm it?

Reply:
@kuiwang02 @jianzhangbjz already set FEATURE_SET for workflow `cucushift-installer-rehearse-aws-ipi-proxy-techpreview-po` on job config level

Reply:
seem like cucushift-installer-rehearse-aws-ipi-proxy-techpreview-po get supported from 4.12+, so I think we can remove `aws/ipi/proxy/techpreview/po/` folder now.